Makefile

```diff
@@ -0,0 +1,131 @@
+# -*- Mode: makefile-gmake -*-
+
+.PHONY: clean all release
+
+#
+# Default target
+#
+
+all: debug release
+
+#
+# Library name
+#
+NAME = sailfish-connman-iptables-plugin
+LIB_NAME = $(NAME)
+LIB_SONAME = $(LIB_NAME).so
+LIB = $(LIB_SONAME)
+
+#
+# Sources
+#
+SRC = \
+ sailfish-iptables-dbus.c \
+ sailfish-iptables-validate.c \
+ sailfish-iptables-parameters.c \
+ sailfish-iptables-utils.c \
+ sailfish-iptables-policy.c \
+ sailfish-iptables.c
+
+#
+# Directories
+#
+
+SRC_DIR = src
+BUILD_DIR = build
+DEBUG_BUILD_DIR = $(BUILD_DIR)/debug
+RELEASE_BUILD_DIR = $(BUILD_DIR)/release
+
+#
+# Tools and flags
+#
+
+CC = $(CROSS_COMPILE)gcc
+LD = $(CC)
+WARNINGS = -Wall
+BASE_FLAGS = -fPIC -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -Wformat -Wformat-security -fmessage-length=0
```

Comment:
Please remove flags that are not absolutely needed. OBS will provide `CFLAGS` environment appropriate for the target platform. I think `-fPIC -fvisibility=hidden` should be enough.

Reply:
OK, will remove.